Dockerfile-ubuntu.template

```diff
@@ -230,6 +230,57 @@ COPY --from=erlang-builder $ERLANG_INSTALL_PATH_PREFIX $ERLANG_INSTALL_PATH_PREF
 COPY --from=openssl-builder $OPENSSL_INSTALL_PATH_PREFIX $OPENSSL_INSTALL_PATH_PREFIX
 ENV PATH $ERLANG_INSTALL_PATH_PREFIX/bin:$OPENSSL_INSTALL_PATH_PREFIX/bin:$PATH
 
+RUN mkdir -p /usr/local/share/sbom/ && \
+<<EOT cat > /usr/local/share/sbom/openssl-erlang.spdx.json
+{{
+	{
+		spdxVersion: "SPDX-2.3",
+		SPDXID: "SPDXRef-DOCUMENT",
+		name: "openssl-erlang-sbom",
+		documentNamespace: "https://docker.com/docker-scout/fs/sbom-61b3df18-3e41-47b8-a954-e4224f48b2f7",
+		dataLicense: "CC0-1.0",
+		packages: [
+			{
+				name: "openssl",
+				versionInfo: .openssl.version,
+				SPDXID: "SPDXRef-Package--openssl",
+				externalRefs: [
+					{
+						referenceCategory: "PACKAGE-MANAGER",
+						referenceType: "purl",
+						referenceLocator: ("pkg:deb/ubuntu/openssl@" + .openssl.version +"?os_name=ubuntu\u0026os_version=" + .ubuntu.version)
+					}
+				]
+			},
+			{
+				name: "erlang",
+				versionInfo: .otp.version,
+				SPDXID: "SPDXRef-Package--erlang",
+				externalRefs: [
+					{
+						referenceCategory: "PACKAGE-MANAGER",
+						referenceType: "purl",
+						referenceLocator: ("pkg:deb/ubuntu/erlang@" + .otp.version +"?os_name=ubuntu\u0026os_version=" + .ubuntu.version)
+					}
+				]
+			},
+			{
+				name: "rabbitmq",
+				versionInfo: .version,
+				SPDXID: "SPDXRef-Package--rabbitmq",
+				externalRefs: [
+					{
+						referenceCategory: "PACKAGE-MANAGER",
+						referenceType: "purl",
+						referenceLocator: ("pkg:github/rabbitmq/rabbitmq-server@" + .version)
```

Comment:
There is an ubuntu package for rabbitmq, https://packages.ubuntu.com/kinetic/rabbitmq-server . There's actually a few since rabbitmq is split into server, lib, dev, and client packages.

Reply:
I wasn't sure on this one because it's not part of the main ubuntu distribution and needs a custom source. I wasn't sure if that was supported

Reply:
Have to checked if there is a generic PURL for packages not associated with a distribution?

Reply:
Perhaps we should be using the `generic` type: https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#generic